itests/direct_data_onboard_test.go

```diff
@@ -293,6 +315,86 @@ func TestOnboardRawPieceVerified(t *testing.T) {
 	allocations, err = client.StateGetAllocations(ctx, verifiedClientAddr, types.EmptyTSK)
 	require.NoError(t, err)
 	require.Len(t, allocations, 0)
+
+	evts, err := miner.FullNode.GetActorEvents(ctx, &types.ActorEventFilter{
+		MinEpoch: -1,
+		MaxEpoch: -1,
+	})
+	require.NoError(t, err)
+
+	for _, evt := range evts {
+		fmt.Printf("Got ActorEvent: %+v", evt)
+	}
+
```

Comment:
from here below we use the existing lotus APIs to get receipts and their event logs for the whole chain, which works